solution/index.js

```diff
@@ -0,0 +1,451 @@
+createPage();
+let liTarget = undefined;
+let sectionid =undefined;
+let prevSection =undefined;
```

Comment:
Don't use global variables. You don't usually need them, and they usually cause trouble. Find a better way to get the relevant elements from the event handler